src/components/shared/inputs/Pay/PayInputSubText.tsx

```diff
@@ -27,27 +25,40 @@ import { V1_CURRENCY_ETH } from 'constants/v1/currency'
  * Else, display the exchange rate of the user selected currency to project token.
  */
 export default function PayInputSubText({
-  payInCurrrency,
-  weiPayAmt,
+  payInCurrency,
+  amount,
+  reservedRate,
+  weight,
+  tokenSymbol,
+  tokenAddress,
 }: {
-  payInCurrrency: V1CurrencyOption
-  weiPayAmt: BigNumber | undefined
+  payInCurrency: number
+  amount: string | undefined
+  reservedRate: number | undefined
+  weight: BigNumber | undefined
+  tokenSymbol: string | undefined
+  tokenAddress: string | undefined
 }) {
-  const { currentFC, tokenSymbol, tokenAddress } = useContext(V1ProjectContext)
   const converter = useCurrencyConverter()
   const {
     theme: { colors },
   } = useContext(ThemeContext)
 
+  const {
+    currencyMetadata,
+    currencies: { currencyETH },
+  } = useContext(CurrencyContext)
+
   const tokenText = tokenSymbolText({
     tokenSymbol: tokenSymbol,
     capitalize: false,
     plural: true,
   })
+  const weiPayAmt = AmountToWei({ currency: payInCurrency, amount: amount })
```

Comment:
**issue** i think hooks should only be within other hooks.

AmountToWei is essentially a hook here.

We could have a hook useWeiConverter that does this, perhaps.

Reply:
Ok yeah good point, I've made a `useWeiConverter` hook which does the same thing as the previous (now deleted) `AmountToWei` util function, is this what you were thinking?